src/Cake.TeamCityCoverageReporter.Tests/TeamCityCoverageReporterTests.cs

```diff
@@ -39,5 +36,22 @@ public async Task GIVEN_SampleFiles_WHEN_ReporterCalled_THEN_OutputIsAsExpected(
             var expected = await File.ReadAllTextAsync(sampleExpectedResultsPath).ConfigureAwait(false);
             result.ShouldBe(expected);
         }
+
+        [Fact]
+        public async Task GIVEN_SampleFiles_WHEN_ReporterCalled_THEN_OutputIsAsExpected()
+        {
+            const string sampleFile = @".\Sample1.xml";
+            const string sampleExpectedResultsPath = @".\Sample1-Expected.txt";
+            await WhenReceiveThenExpectTeamCityCoverageReport(sampleFile, sampleExpectedResultsPath);
```

Comment:
Just as a note here you can replace the await with return and remove the async from the method. Doing so reduces some of the overhead of async methods (only really matters if being called thousands of times)